src/rp2040.ts

```diff
@@ -1688,27 +1690,21 @@ export class RP2040 {
     else if (opcode === 0b1011111100100000) {
       // do nothing for now. Wait for event!
       this.cycles++;
-      if (this.logMissingInstructions) {
-        console.log('WFE');
-      }
+      this.logger.info(LOG_NAME, "WFE");
     }
     // WFI
     else if (opcode === 0b1011111100110000) {
       // do nothing for now. Wait for event!
       this.cycles++;
-      if (this.logMissingInstructions) {
-        console.log('WFI');
-      }
+      this.logger.info(LOG_NAME, "WFI");
```

Comment:
Hmm, this doesn't seem like it was properly formatted with prettier, as our prettier configuration requires single quote single ('string' and not "string")

Reply:
Will fix manually, prettier has been run.

Reply:
Done